### PR TITLE
Support room type

### PR DESCRIPTION
**Pull request by timmydoza**
**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

https://issues.corp.twilio.com/browse/AHOYAPPS-828

This PR adds support for choosing room type when deploying video apps.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.3.0
+
+### New Feature
+
+- This version adds the ability to specify the room type when deploying a video application token server. Use the `--room-type` flag when deploying the app. For example: `twilio rtc:apps:video:deploy --authentication passcode --room-type group-small`. The payload returned by the token server will also include the room type (see the [readme](README.md) for details).
```
Comment (aaalaniz) [inline CHANGELOG.md:7]: Add statement about the default value and that the new argument is optional

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@twilio-labs/plugin-rtc",
-  "version": "0.1.7",
+  "version": "0.3.0",
```
Comment (aaalaniz) [inline package.json:3]: Oh weird. I think we need to submit a PR to the current master branch that updates the version to 0.2.0

File: README.md [hunk 3]
```diff
@@ -108,7 +108,8 @@ POST /token
 
 ```json
```
Comment (aaalaniz) [inline README.md:109]: I think we agreed to add `create_room` to JSON POST payload that allows user to skip creating the room right?

Comment (timmydoza) [reply README.md:109]: Oh right. Was it `adhoc: true` that would skip room creation?

File: test/e2e/e2e.test.js [hunk 4]
```diff
@@ -116,16 +121,39 @@ describe('the RTC Twilio-CLI Plugin', () => {
...
-        expect(stdout.output).toMatch(/Web App URL: .+\nPasscode: \d{3} \d{3} \d{4} \d{4}\nExpires: .+/);
+        expect(stdout.output).toMatch(
+          /Web App URL: .+\nPasscode: \d{3} \d{3} \d{4} \d{4}\nExpires: .+\nRoom Type: group/
+        );
       });
     });
 
-    describe('the deploy command', () => {
-      it('should return a video token when the correct passcode is provided', async () => {
+    describe('the serverless deployment', () => {
```
Comment (aaalaniz) [inline test/e2e/e2e.test.js:130]: Can we add a test that validates the token is properly returned when the REST create fails?

File: README.md [hunk 5]
```diff
@@ -172,6 +174,19 @@ POST /token
...
 
+<tr>
+<td> 400 </td>
+<td>
+
+```json
+{
+  "error": {
+    "message": "invalid parameter",
+    "explanation": "A boolean value must be provided for the create_room parameter"
```
Comment (timrozum) [inline README.md:185]: But `create_room` is optional right? So this error would never happen?

Comment (aaalaniz) [reply README.md:185]: @timrozum if you look at the tests below you'll see that this error covers cases where `create_room` is passed `'a string'` for example.

Comment (timrozum) [reply README.md:185]: Oh I see thanks!

**Pull request merged**